app/src/Repository/MsgrcptSearchRepository.php

```diff
@@ -15,7 +15,7 @@
 class MsgrcptSearchRepository extends MsgrcptRepository
 {
     /**
-     * @param ?array{sort: string, direction: string} $sortParams
+     * @param array{}|array{sort: string, direction: string} $sortParams
```

Comment:
Ça me semblait bien avant, on ne gère pas le cas où `$sortParams` est vide

```suggestion
     * @param ?array{sort: string, direction: string} $sortParams
```